src/components/ContributionsEpic.stories.tsx

```diff
@@ -12,10 +13,12 @@ export default {
 
 export const defaultStory = (): ReactElement => {
     // Epic content props
-    const epicContent = {
+    const variant: Variant = {
+        name: 'Test Epic',
         heading: text('heading', testData.content.heading),
         paragraphs: object('paragraphs', testData.content.paragraphs),
-        highlighted: object('highlighted', testData.content.highlighted),
+        highlightedText: text('highlightedText', testData.content.highlightedText),
+        showTicker: false,
```

Comment:
I've intentionally not made this editable, since we don't yet support it in the component.